tests/test_input_manager.py

```diff
@@ -4779,3 +4809,254 @@ def test_extract_input_data_by_key_list_key_error(
         var_name=var_name,
         called_during_initialization=called_during_initialization,
     )
+
+
+@pytest.fixture
+def mock_metadata_prepare_data() -> dict[Any, Any]:
+    return {
+        "properties": {
+            "example_blob_key": {
+                "root": {"nested_property": {"type": "string", "description": "An example property"}},
```

Comment:
```suggestion
                "object_property": {"nested_property": {"type": "string", "description": "An example property"}},
```
Root is a pretty loaded term, I think it would be better to use a different term because the test cases get kind of confusing to read when there is a "root" present in them.